packages/marko-web-inquiry/i18n.js

```diff
@@ -0,0 +1,12 @@
+import translations from './translations';
+
+export default (lang, fieldLabelKey) => {
+  const keys = Object.keys(translations);
+  if (!keys.includes(lang)) {
```

Comment:
One of my comments got lost! You don't need to use Object.keys here -- it's an object, so just check if the key is truthy using array syntax (`translations[lang]`) instead of converting to an array to check if the key is present.